.github/workflows/build_tool.yaml

```diff
@@ -40,3 +40,6 @@ jobs:
           context: tool/${{ matrix.image }}
           tags: ${{ env.REGISTRY }}/pipe-cd/${{ matrix.image }}:${{ env.PIPECD_VERSION }}
           platforms: linux/amd64,linux/arm64
+          # parameter to use inline cache. ref; https://docs.docker.com/build/ci/github-actions/cache/#inline-cache
+          cache-from: type=registry,ref=${{ env.REGISTRY }}/pipe-cd/${{ matrix.image }}:latest
+          cache-to: type=inline
```

Comment:
[IMO] How about adding `mode=max` ? 
It seems that we can cache all layer with it. 
https://docs.docker.com/build/cache/backends/#cache-mode

It might be more efficient because our images are often built with mutl-stage build to download dependencies.

Reply:
Unfortunately, the inline cache doesn't support the max cache mode.
> However, note that the inline cache exporter only supports min cache mode.

ref; https://docs.docker.com/build/ci/github-actions/cache/#inline-cache

We can consider using the GitHub cache, but it's experimental yet. So, I want to start by using inline cache and min cache mode.

Reply:
I got it. Thanks!